### PR TITLE
Test enstore plotter module.py

### PR DESCRIPTION
**Pull request by dbox-fnal**

### CODE REVIEW

File: src/enstore_plotter_module.py [hunk 1]
```diff
@@ -62,8 +108,13 @@ def roundtime(seconds, rounding=None):
     """
     Round the provided time and return it.
 
-    `seconds`: time in seconds.
-    `rounding`: None, 'floor' or 'ceil'.
+    Args:
+        seconds: time in seconds.
+        rounding: None, 'floor' or 'ceil'.
+    Returns:
+        time in seconds with indicated rounding applied .
```
Comment (renbauer-fnal) [inline src/enstore_plotter_module.py:113]: So does this function do nothing if no `rounding` argument is supplied?

File: src/enstore_plotter_module.py [hunk 2]
```diff
@@ -12,30 +12,76 @@
...
-#import enstore_plotter_framework
 
 class EnstorePlotterModule:
+    """Base class for all Enstore plotter modules."""
     def __init__(self,name,isActive=True):
         self.name=name
         self.is_active=isActive
         self.parameters = {}
+    
     def isActive(self):
```
Comment (renbauer-fnal) [inline src/enstore_plotter_module.py:23]: isActive and setActive seem to do real things, is it correct to say they're implemented by subclasses?

File: src/tests/test_enstore_plotter_module.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    def test___init__(self):
+        self.assertTrue(self.epm.name == "test")
+        self.assertTrue(isinstance(self.epm, enstore_plotter_module.EnstorePlotterModule))
+
+    def test_set_and_isActive(self):
+        self.assertEqual(self.epm.isActive(), True) 
+        self.epm.setActive(False)
+        self.assertEqual(self.epm.isActive(), False)
+
+    def test_book(self):
```
Comment (renbauer-fnal) [inline src/tests/test_enstore_plotter_module.py:22]: These four look a bit strange, I guess they're just testing that the functions they "test" don't raise errors, but that seems like the opposite behavior from what I would expect for functions that are intended to be overwritten by subclasses -- I think the "correct" think to do would be to throw errors in these functions and test for that, but it is probably not worth adding failure cases to the codebase... 

I think what might make it a bit more clear would be to change all these functions (book, fill, plot, install) to simply `pass` and comment in the tests as well that they're all noops intended to be overwritten. WDYT?

**Review by renbauer-fnal — APPROVED** [review]
A few minor comments - I'm OOO next week so I'm approving in advance. Please feel free to merge without additional review after addressing comments.

File: src/enstore_plotter_module.py (hunk 1 above)
Comment (dbox-fnal) [reply src/enstore_plotter_module.py:113]: That appears to be the case.  This function only seems to be called with 'ceil' in the code

File: src/enstore_plotter_module.py (hunk 2 above)
Comment (dbox-fnal) [reply src/enstore_plotter_module.py:23]: Good catch.  Thank you.

**Pull request merged**